sigs.yaml

```diff
@@ -1070,6 +1070,7 @@ sigs:
       mailing_list: https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/cluster-api/main/OWNERS
+    - https://raw.githubusercontent.com/kubernetes-sigs/karpenter-provider-cluster-api/refs/heads/main/OWNERS
```

Comment:
this should be under a new subproject called with the same name as the repo name, instead of enumerating the new OWNERS file under an existing subproject cluster-api.

cc @elmiko

Reply:
^ please update if possible @Priyankasaggu11929

Reply:
@neolit123, updated.